libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -262,14 +277,224 @@ class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettings
     return featureFlagValue;
   }
 
-  private getLowerCaseFilters(clientFilter: ClientFilter): { [key: string]: any } {
-    const keys: Array<string> = Object.keys(clientFilter);
+  private getFeatureFlagsFromV2Schema(featureFlag: any): FeatureFlagValue {
+    const featureFlagValue: FeatureFlagValue = {
+      id: "",
+      description: "",
+      enabled: false,
+      conditions: {
+        clientFilters: []
+      }
+    };
+
+    featureFlagValue.id = featureFlag.id;
+    featureFlagValue.enabled = featureFlag.enabled.toLowerCase() == "true";
+    featureFlagValue.description = featureFlag.description || "";
+    if (featureFlag.telemetry) {
+      featureFlagValue.telemetry = featureFlag.telemetry;
+    }
+
+    const clientFilters = featureFlag.conditions && featureFlag.conditions[this.featureFlagsKeyWordsCase.client_filters];
+    if (clientFilters && clientFilters.length != 0) {
+      featureFlagValue.conditions.clientFilters = this.validateClientFilters(featureFlagValue.id, "ffv2", clientFilters);
+    }
+
+    const requirementType = featureFlag.conditions && featureFlag.conditions[this.featureFlagsKeyWordsCase.requirement_type];
+    if (requirementType) {
+      if (this.validateRequirementType(requirementType, featureFlag.id)) {
+        featureFlagValue.conditions.requirement_type = requirementType;
+      }
+    }
+
+    if (featureFlag.allocation) {
+      featureFlagValue.allocation = {};
+      const default_when_disabled = featureFlag.allocation[this.featureFlagsKeyWordsCase.default_when_disabled];
+      if (default_when_disabled) {
```

Comment:
Could we rename `featureFlagsKeyWordsCase` to `featureFlagKeyWords` since the variable holds the feature flags key words ?